### DISCUSSION TITLE
LX06 - what kind of charger?

### DISCUSSION THREAD

**Discussion by Dinth** [duhow/xiaoai-patch#84] [General]
Hi. A while ago i have purchased LX06 shipped from China, but since it has arrived with some useless (for me) plug charger i have disposed the charger. Unfortunately now i have no idea what charger i should purchase for it, except its 12v/2A and some kind of barrel - but seems like non-standard small barrel.

**Comment by duhow** [comment]
If I recall correctly, should be Tuning Fork Connector 4.0*1.7mm
